### PR TITLE
Update minecraft launcher url

### PR DESCRIPTION
**Pull request by Jerzerak**

### CODE REVIEW

File: Casks/minecraft.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   sha256 :no_check
 
   # amazonaws.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/minecraft.rb]: Change to `# mojang.com was verified as official when first introduced to the cask`.

**Comment by Jerzerak** [comment]
Just realized, would you like the commits squashed or are they fine as is?

**Comment by vitorgalvao** [comment]
We always prefer them squashed.

**Comment by Jerzerak** [comment]
Squashed as requested

**Pull request merged**